cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -1107,6 +1114,8 @@ private void deleteZnodesToStopExecution() {
           LOG.debug("Tasks in execution: {}", _executionTaskManager.inExecutionTasks());
         }
         List<ExecutionTask> deadOrAbortingTasks = new ArrayList<>();
+        List<ExecutionTask> slowTasksToReport = new ArrayList<>();
+        boolean shouldReportSlowTasks = _time.milliseconds() - _lastSlowTaskReportingTimeMs > SLOW_TASK_ALERTING_BACKOFF_TIME_MS;
```

Comment:
Which code path updates the value of `_lastSlowTaskReportingTimeMs`? 
-- i.e. It seems it will always be `-1`.

Reply:
Good catch!